dhcpv4/option_vivc.go

```diff
@@ -0,0 +1,107 @@
+package dhcpv4
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"sort"
+)
+
+// This option implements the Vendor-Identifying Vendor Class Option
+// https://tools.ietf.org/html/rfc3925
+
+// OptVIVC represents the DHCP message type option.
+type OptVIVC struct {
+	Identifiers map[uint32][]byte
```

Comment:
For how much I like the consistency that comes from using a map instead of a list, I would still prefer to use a slice here.
In my view, a sequence of bytes should be decoded and re-encoded, and yield exactly the same bytes, even if they generate the same broken packet or option as in origin. The validation, if desired, could be done somewhere else

Reply:
Makes sense.  I will go ahead and update this.